src/client/components/illustrated-content/readme.md

```diff
@@ -0,0 +1,2 @@
+# Illustrated content
+
```

Comment:
echooooooo

shouldn't you put some text here? :-P

Reply:
I updated the text